ui/src/app/dashboard/grouped-summary/grouped-summary.component.html

```diff
@@ -1,31 +1,45 @@
-<mat-card>
-  <mat-card-title>
-    {{ aggregation.name }}
-  </mat-card-title>
+<div [class]="cardClass" id="container">
+  <mat-card>
+    <mat-card-content>
+      <table matSort (matSortChange)="sortData($event)">
+        <tr>
+          <th mat-sort-header="label" class="label">{{ aggregation.name }}</th>
```

Comment:
The sort arrow looks really far from the status icons. Do you have any control over that placement?